code/game/objects/structures/doors.dm

```diff
@@ -8,6 +8,8 @@
 #define DOOR_OPENING 1<<3
 ///Is currently closing
 #define DOOR_CLOSING 1<<4
+///Resin door has special properties such as ignoring intent and not being slammed shut
```

Comment:
And shouldn't you split this into the constituent behaviors?

Reply:
Would have thought someone would say it was bloaty so put it as one for anything resin-related.